cli/cmd/dashboard.go

```diff
@@ -23,45 +23,35 @@ var dashboardCmd = &cobra.Command{
 			log.Fatalf("port must be positive, was %d", proxyPort)
 		}
 
-		kubectl, err := k8s.NewKubectl(shell.NewUnixShell())
+		api, err := k8s.NewK8sAPI(shell.NewUnixShell().HomeDir(), kubeconfigPath)
 		if err != nil {
-			log.Fatalf("Failed to start kubectl: %v", err)
+			log.Fatalf("NewK8sAPI() failed with: %+v", err)
 		}
 
-		asyncProcessErr := make(chan error, 1)
-
-		err = kubectl.StartProxy(asyncProcessErr, proxyPort)
-
+		url, err := api.ProxyUrlFor(controlPlaneNamespace, "/services/web:http/proxy/", proxyPort)
 		if err != nil {
-			log.Fatalf("Failed to start kubectl proxy: %v", err)
-		}
-
-		url, err := kubectl.UrlFor(controlPlaneNamespace, "/services/web:http/proxy/")
-
-		if err != nil {
-			log.Fatalf("Failed to generate URL for dashboard: %v", err)
+			log.Fatalf("Failed to generate URL for dashboard: %+v", err)
```

Comment:
Should we at some point have a blanket PR adding `+` to all `%v`?

Reply:
Hmm, yeah, we use this pattern all over the place, but my preference is to use "%s" when printing errors. We don't gain anything with "%v" or "%+v", and those aren't type checked by `go vet`. I'd be 👍 👍 👍  for a branch that converts all printed errors to "%s". https://play.golang.org/p/KqJ4PezUhGs

Reply:
Hmm, I generally lean towards blanket `%+v` everywhere, particularly if `%s` and `%+v` yield the same output for `error` types, it's one less thing for me to think about. But, I had not considered the `go vet` angle.